modules/openapi-generator/src/main/resources/dart2/api_client.mustache

```diff
@@ -147,9 +146,48 @@ class ApiClient {
 
     throw ApiException(HttpStatus.badRequest, 'Invalid HTTP operation: $method $path',);
   }
+{{#native_serialization}}
+
+  Future<dynamic> deserializeAsync(String json, String targetType, {bool growable}) async =>
+    // ignore: deprecated_member_use_from_same_package
+    deserialize(json, targetType, growable: growable);
+
+  @Deprecated('Scheduled for removal in 6.0. Use deserializeAsync() instead.')
```

Comment:
minor suggestion: replace `6.0` with `OpenAPI Generator 6.x`

as users may have no idea what `6.0` refers to